openapi-core/src/main/java/io/ballerina/openapi/core/generators/client/FunctionBodyGenerator.java

```diff
@@ -662,30 +658,43 @@ private void createRequestBodyStatements(boolean isHeader, List<StatementNode> s
             statementsList.add(expressionStatementNode);
         }
         // POST, PUT, PATCH, DELETE, EXECUTE
-        VariableDeclarationNode requestStatement =
-                GeneratorUtils.getSimpleStatement(returnType, RESPONSE, "check self.clientEp->"
-                        + method + "(" + RESOURCE_PATH + ", request)");
+        String requestStatement = "check self.clientEp->" + method + "(" + RESOURCE_PATH + ", request)";
```

Comment:
`formatted()` util in java17 will look cleaner in the above context IMO. WDYT? 
```suggestion
        "check self.clientEp->%s(%s, request)".formatted(method, RESOURCE_PATH);
```

Reply:
Yeah , I will update with the suggested approach